src/containers/SketchParam/index.js

```diff
@@ -11,6 +12,9 @@ const mapStateToProps = (state, ownProps) => ({
 const mapDispatchToProps = (dispatch, ownProps) => ({
   onAdvancedClick: () => {
     dispatch(uiAuxToggleOpen(ownProps.nodeId))
+    setTimeout(() => {
+      uiEventEmitter.emit('recalc-param-heights')
```

Comment:
The way the param boxes are styled is a bit strange and when they open/close and can cause overlapping of elements when heights change. The UI needs to be told that heights have changed. The way I've been handling this is with a little event emitter.